landoui/pages.py

```diff
@@ -46,8 +46,7 @@ def home():
 def protected():
     logger.warn(
         {
-            'user_name': session['userinfo']['name'],
-            'email': session['userinfo']['email']
```

Comment:
Email was causing an error when I log in via Facebook, so I just removed it.  This is just a sample usage anyway.